psi4/src/psi4/dfocc/ccsd_tpdm.cc

```diff
@@ -31,60 +31,59 @@
 #include "defines.h"
 #include "dfocc.h"
 
-
 using namespace psi;
 
 namespace psi{ namespace dfoccwave{
-
+  
 void DFOCC::ccsd_tpdm()
-{
+{   
 
     timer_on("tpdm");
     SharedTensor2d T, U, Tau, G, G2, V, X, Y, Z;
-    SharedTensor2d Ts, Ta, Vs, Va, S, A;
+    SharedTensor2d Ts, Ta, Vs, Va, S, A, K;
 
 //if (reference_ == "RESTRICTED") {
-
+    
     //============================
     // OO-Block Correlation TPDM
     //============================
 
-    // G_ij^Q = P+(ij) 2*\cal(V)_ij^Q
+    // G_ij^Q = P+(ij) 2*\cal(V)_ij^Q 
     G = SharedTensor2d(new Tensor2d("Correlation 3-Index TPDM (Q|IJ)", nQ, naoccA, naoccA));
     V = SharedTensor2d(new Tensor2d("calV (Q|IJ)", nQ, naoccA, naoccA));
     V->read(psio_, PSIF_DFOCC_AMPS);
     G->axpy(V, 2.0);
     V.reset();
-    // G_ij^Q += P+(ij) V_ij^Q
+    // G_ij^Q += P+(ij) V_ij^Q 
     V = SharedTensor2d(new Tensor2d("V (Q|IJ)", nQ, naoccA, naoccA));
     V->read(psio_, PSIF_DFOCC_AMPS);
     G->axpy(V, 1.0);
     V.reset();
-    // G_ij^Q += P+(ij) Vt_ij^Q
+    // G_ij^Q += P+(ij) Vt_ij^Q 
```

Comment:
There's a nifty format file to drive `clang-format` now. It'll get rid of unnecessary whitespace additions, I hope. And uneven indentation

Reply:
The following revisions were made based on Lori's comments:

* DFOCC module I run clang-format
* using namespace std; is deleted from mp3_W_intr.cc file
* proc is revised, now we can run analytic gradients with both ccenergy and dfocc 

@loriab

Reply:
Thanks, @bozkaya, but I don't think the clang-format worked. I tried it locally (`clang-format -i file.cc` on a psi4 repo with `.clang-format` in top-level) on one of your files, and it did get rid of the spaces at end-of-line and uneven indentation. So maybe give it another try.